.gitignore

```diff
@@ -26,4 +26,4 @@ nbproject
 
 # Folders to ignore
 node_modules
-build
+dist
```

Comment:
If we remove the bundled script from the repo, we need to add the rollup scripts to `postinstall` in package.json, and move rollup etc. from `devDependencies` to `dependencies`, right?

Reply:
No, the user should never have to bundle a module. Your `prepublish` already handles bundling your app and creating the "dist” folder prior to publishing. `npm publish` publishes everything in your directory (if its not excluded by a .npmignore file), so it will always include the "dist" folder.

This will work locally and also inside a pipeline.

Reply:
"If there’s no .npmignore file, but there is a .gitignore file, then npm will ignore the stuff matched by the .gitignore file." – from https://docs.npmjs.com/using-npm/developers.html#description
I guess you could also add an empty .npmignore file?

Reply:
ok, thats new to me...  `npm publish --dry-run` lists the `dist/*` files in Tarball Contents... Nevertheless I've added an empty .npmignore file.